eng/Packages.Data.props

```diff
@@ -89,7 +89,7 @@
     <PackageReference Update="System.ClientModel" Version="1.2.1" />
     <PackageReference Update="System.IO.Hashing" Version="6.0.0" />
     <PackageReference Update="System.Memory" Version="4.5.5" />
-    <PackageReference Update="System.Memory.Data" Version="6.0.0" />
+    <PackageReference Update="System.Memory.Data" Version="8.0.0" />
```

Comment:
This cannot happen at the moment due to incompatibilities with Azure Functions and PowerShell hosts.    

Once `net6.0` is formally retired by these platforms on Nov 12, we'll look at bumping up to the 8.x range for dependencies, working with the Functions team to negotiate any common versions.

Reply:
I need access to BinaryData.MediaType property. Is there any other way to acomplish that?

Reply:
I did a local override